sdk/storage/Azure.Storage.Files.DataLake/src/Models/DataLakeLease.cs

```diff
@@ -6,22 +6,25 @@
 namespace Azure.Storage.Files.DataLake.Models
 {
     /// <summary>
-    /// Lease
+    /// Lease.
     /// </summary>
     public class DataLakeLease
     {
         /// <summary>
-        /// The ETag contains a value that you can use to perform operations conditionally. If the request version is 2011-08-18 or newer, the ETag value will be in quotes.
+        /// The ETag contains a value that you can use to perform operations conditionally.
+        /// If the request service version is 2011-08-18 or newer, the ETag value will be in quotes.
```

Comment:
I thought the SDKs stripped quotes so they would all be the same. Is this not the case?

Reply:
Etag.ToString() returns the ETag with quotes.